src/worldofzuul/Character.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Kasper
+ */
+public class Character {
+    private String name;
```

Comment:
add docblock for name attribute

Reply:
Fixed in 2c64b148ac86de64b48b13ee22f6b153531114b3